test/dummy/GameMotionChild.cpp

```diff
@@ -0,0 +1,9 @@
+/**
+ * @file GameMotionChild.cpp
+ * @brief GameMotionの子クラス
```

Comment:
```suggestion
 * @brief GameMotionの子クラス（ダミー）
```
他のダミークラスでは明記されているので合わせた方がいいと思います。